src/features/import-assertions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Import Assertions'
+author: 'Dan Clark ([@dandclark1](https://twitter.com/dandclark1))'
+avatars:
+  - 'dan-clark'
+date: 2021-06-10
+tags:
+  - ECMAScript
+description: 'Import assertions allow module import statements to include additional information alongside the module specifier'
+tweet: ''
+---
+
+The new [import assertions](https://github.com/tc39/proposal-import-assertions) feature allows module import statements to include additional information alongside the module specifier. An initial use for the feature is to enable JSON documents to be imported as [JSON modules](https://github.com/tc39/proposal-json-modules):
+
+```javascript
+// foo.json
+{ "answer": 42 }
+
+// main.mjs
+import json from "./foo.json" assert { type: "json" };
```

Comment:
Nit: we use single-quoted in our codestyle.

(we really should automate this...)